src/frontend/frontend/file.js

```diff
@@ -58,16 +58,16 @@ angular.module('frontend-app')
          *  the socket exists in the first place, this is
          *  fine for now. */
         var cbC_key = ws.register_callback('connected', function () {
-          if (self.editor)
```

Comment:
Note: when we go online, we'll have to reload contents/undo history, as the contents/undo history of the file might have changed (due to a conflict).